Makefile

```diff
@@ -21,61 +26,125 @@ help:  ## Display this help
 				printf "\n%s%s%s\n", col, substr($$0, 5), nocol \
 			} \
 		' $(MAKEFILE_LIST)
+###############################################################################
+
+################################ make install #################################
+.PHONY: install
+install: ## Installs all dependencies needed to compile Scorecard
+install: | $(GINKGO) $(GOLANGGCI_LINT) $(PROTOC_GEN_GO) $(PROTOC)
+
+$(GINKGO):
+	go get -u github.com/onsi/ginkgo/ginkgo@v1.16.2
+
+$(GOLANGGCI_LINT):
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.40.0
+
+$(PROTOC_GEN_GO):
+	go install google.golang.org/protobuf/cmd/protoc-gen-go
```

Comment:
Can you please pin the dependency version for security reasons, similar to others.

Reply:
Here is the issue to track this https://github.com/ossf/scorecard/issues/459